docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,442 @@
+<2> We calculate the training error by only evaluating the training data.
+<3> The field that contains the ground truth label.
+<4> The field that contains the predicted value.
+
+Next, we calculate the generalization error that represents how well the model 
+performed on previously unseen data:
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/_evaluate
+{
+ "index": "df-flight-delayed",
+   "query": {
+    "term": {
+      "ml.is_training": {
+        "value": false  <1>
+      }
+    }
+  },
+ "evaluation": {
+   "classification": {
+     "actual_field": "FlightDelay",
+     "predicted_field": "ml.FlightDelay_prediction",
+     "metrics": {  
+       "multiclass_confusion_matrix" : {}
+     }
+   }
+ }
+}
+--------------------------------------------------
+// TEST[skip:TBD]
+
+<1> We evaluate only the documents that are not part of the training data.
+
+
+The returned confusion matrix shows us how many datapoints were classified 
+correctly (where the `actual_class` matches the `predicted_class`) and how many 
+were misclassified (`actual_class` does not match `predicted_class`):
+
+[source,console-result]
+--------------------------------------------------
+{
+  "classification" : {
+    "multiclass_confusion_matrix" : {
+      "confusion_matrix" : [
+        {
+          "actual_class" : "false", <1>
+          "actual_class_doc_count" : 8778, <2>
+          "predicted_classes" : [
+            {
+              "predicted_class" : "false", <3>
+              "count" : 7509 <4>
+            },
+            {
+              "predicted_class" : "true",
+              "count" : 1269
+            }
+          ],
+          "other_predicted_class_doc_count" : 0
+        },
+        {
+          "actual_class" : "true",
+          "actual_class_doc_count" : 2939,
+          "predicted_classes" : [
+            {
+              "predicted_class" : "false",
+              "count" : 1213
+            },
+            {
+              "predicted_class" : "true",
+              "count" : 1726
+            }
+          ],
+          "other_predicted_class_doc_count" : 0
+        }
+      ],
+      "other_actual_class_count" : 0
+    }
+  }
+}
+--------------------------------------------------
+<1> The name of the actual class. In this example, there are two actual classes: 
+`true` and `false`.
+<2> The number of documents in the dataset that belong to the actual class.
+<3> The name of the predicted class.
+<4> The number of documents belong to the actual class that are labelled as the 
+predicted class. 
+
+There are 8778 documents in the testing proportion of the dataset that have the 
+`false` class. The model labelled 7509 documents (out of 8778) correctly as 
+`false` and 1269 documents as `true` while those are actually `false`. There are 
+2939 documents in the testing data that have the `true` class. 1213 of them are 
+predicted as `false` and 1726 are predicted correctly as `true`.
```

Comment:
Because it is only a binary classification example, I think it is worth additionally representing this data as tabular confusion matrix not just in words. 

Also, the sample data changes when you load it.. Please check if the values change or if it is just the timestamps. We do need the check this, because running the same analysis against different instances of Kibana where the flight data has been generated differently may result in different answers. In this case, we should qualify the results may vary.